mkdocs.yml

```diff
@@ -86,7 +87,8 @@ nav:
         - Replicas Recommendation: tutorials/replicas-recommendation.md
       - Qos Ensurance: tutorials/using-qos-ensurance.md
```

Comment:
Where is the menu for dynamic-scheduler-plugin?

Reply:
@qmhu already fixed